src/components/about-offer/AboutOffer.jsx

```diff
@@ -0,0 +1,44 @@
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import Typography from '@mui/material/Typography'
+import { useTranslation } from 'react-i18next'
+import aboutOfferTitle from '~/constants/translations/en/common'
+import showMore from '~/constants/translations/en/common'
+import showLess from '~/constants/translations/en/common'
```

Comment:
Pay attention to the path, it is the same. So here we are importing the same file for three times.
Instead we can import the file only once:
```javascript
import common from '~/constants/translations/en/common'
```
and then use: `common.aboutOffer`, `common.showMore` and so on.

But to work with translations we don't even need to import a json file.